### PR TITLE
Create TRANSPARENCY_FAQS.md

### PR DESCRIPTION
**Pull request by markwallace-microsoft**
### Contribution Checklist

- [ ] The code builds clean without any errors or warnings
- [ ] The PR follows the [SK Contribution Guidelines](https://github.com/microsoft/semantic-kernel/blob/main/CONTRIBUTING.md) and the [pre-submission formatting script](https://github.com/microsoft/semantic-kernel/blob/main/CONTRIBUTING.md#development-scripts) raises no violations
- [ ] All unit tests pass, and I have added new tests where possible
- [ ] I didn't break anyone :smile:

### CODE REVIEW

File: TRANSPARENCY_FAQS.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+Microsoft Semantic Kernel is a lightweight, open-source development kit designed to facilitate the integration of AI models into applications written in languages such as C#, Python, or Java.
+
+It serves as efficient middleware that supports developers in building AI agents, automating business processes, and connecting their code with the latest AI technologies. Input to this system can range from text data to structured commands, and it produces various outputs, including natural language responses, function calls, and other actionable data.
+
+
+## What can Microsoft Semantic Kernel do?
+Building upon its foundational capabilities, Microsoft Semantic Kernel facilitates several functionalities:
+-	AI Agent Development: Users can create agents capable of performing specific tasks or interactions based on user input.
+-	Function Invocation: It can automate code execution by calling functions based on AI model outputs.
+-	Modular and Extensible: Developers can enhance functionality through plugins and a variety of pre-built connectors, providing flexibility in integrating additional AI services.
```
Comment (RogerBarret0) [inline TRANSPARENCY_FAQS.md:13]: Add Filtering and Templating as components that can be leveraged ?

File: TRANSPARENCY_FAQS.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+
+## What are the limitations of Microsoft Semantic Kernel?
+Limitations include:
+-	Contextual Misunderstanding: The system may struggle with nuanced requests, particularly those involving complex context.
+-	Bias in LLM Outputs: Historical biases in the training data can inadvertently influence model outputs. 
+	-	Users can mitigate these issues by:
+		-	Formulating clear and explicit queries.
+		-	Regularly reviewing AI-generated outputs to identify and rectify biases or inaccuracies.
+        -   Providing relevant information when prompting the LLM so that it can base it's responses on this data
```
Comment (westey-m) [inline TRANSPARENCY_FAQS.md:38]: These limitations are really limitations of the LLMs.  Should we call out that Semantic Kernel is limited by the LLMs it's using, and those limitations include these?

**Review by westey-m — APPROVED** [review]

File: TRANSPARENCY_FAQS.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+Building upon its foundational capabilities, Microsoft Semantic Kernel facilitates several functionalities:
+-	AI Agent Development: Users can create agents capable of performing specific tasks or interactions based on user input.
+-	Function Invocation: It can automate code execution by calling functions based on AI model outputs.
+-	Modular and Extensible: Developers can enhance functionality through plugins and a variety of pre-built connectors, providing flexibility in integrating additional AI services.
+-	Multi-Modal Support: The kernel easily expands existing applications to support modalities like voice and video through its architecture
+
+
+## What is/are Microsoft Semantic Kernel’s intended use(s)?
+The intended uses of Microsoft Semantic Kernel include:
+- 	Enterprise Applications: Building scalable enterprise-grade solutions that can leverage advanced AI models for various business applications.
```
Comment (RogerBarret0) [inline TRANSPARENCY_FAQS.md:19]: Enterprise + SMBs.

```suggestion
- 	Production Ready Applications: Building small to large enterprise scale applications that intend ...
```

File: TRANSPARENCY_FAQS.md (hunk 2 above)
Comment (RogerBarret0) [inline TRANSPARENCY_FAQS.md:38]: Those are limitations on the Models, not necessarily in SK. 

Our limitations today are more related to things that we haven't deployed yet but are in the radar like:
- Different modalities that we still don't support, like `Video` or `Classification` and some other that we might want (i.e: HuggingFace/Azure AI Studio inference types)
- Small/Local Models function calling
- Changing Services from Filters
- Other providers connectors

Comment (evchaki) [reply TRANSPARENCY_FAQS.md:38]: Thanks for the call outs, I think we can add these in.  The LLMs do have limits as called out above and that is the main connector most people use, so it is a good one to have in the doc.

File: TRANSPARENCY_FAQS.md [hunk 4]
```diff
@@ -16,7 +16,7 @@ Building upon its foundational capabilities, Microsoft Semantic Kernel facilitat
 
 ## What is/are Microsoft Semantic Kernel’s intended use(s)?
 The intended uses of Microsoft Semantic Kernel include:
-- 	Enterprise Applications: Building scalable enterprise-grade solutions that can leverage advanced AI models for various business applications.
+- 	Production Ready Applications: Building small to large enterprise scale applications that intend ...
```
Comment (RogerBarret0) [inline TRANSPARENCY_FAQS.md:19]: My suggestion was not final in the update, see if this ending works best.

```suggestion
- 	Production Ready Applications: Building small to large enterprise scale solutions that can leverage advanced AI models capabilities
```

**Pull request merged**